js/views/playingScreen.js

```diff
@@ -66,18 +64,6 @@ var PlayingScreen = React.createClass({
     }
   },
 
-  handlePlayerMouseUp: function(event) {
```

Comment:
Why are we removing this function? Won't this effect the main player?  I don't see it being moved anywhere?